plugins/outputs/file/file.go

```diff
@@ -21,8 +23,10 @@ type File struct {
 
 var sampleConfig = `
   ## Files to write to, "stdout" is a specially handled file.
+  ## Will also accept fromats such as /tmp/{020106}/metrics{020106.150406}.out
```

Comment:
you will need to document that better and do a full writeup in the README with examples.